sdk/core/azure-core-test/inc/azure/core/test/test_base.hpp

```diff
@@ -339,6 +339,9 @@ namespace Azure { namespace Core { namespace Test {
     /**
      * @brief Run after each test
      *
+     * @note: If a test case overrides the TearDown method, it MUST call the `TestBase::TearDown`
+     * method, or test recordings will fail to be generated.
```

Comment:
thank you ❤️

Reply:
That was 2 hours of my life that would have been saved if this comment had existed.